common/windows/delphi/ext/sentry/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+builder_describe "Sentry component for Delphi" clean configure build test
+builder_parse "$@"
+
+#-------------------------------------------------------------------------------------------------------------------
+
+source "$KEYMAN_ROOT/resources/build/win/environment.inc.sh"
+
+builder_describe_outputs \
+  configure:project    /resources/build/win/delphi_environment_generated.inc.sh
+
+#-------------------------------------------------------------------------------------------------------------------
+
+function do_test() {
+  # TODO: this does not currently seem to be tested or used, and no longer
+  #       builds but this can be fixed in the future
```

Comment:
Is this tracked somewhere?  Just highlighting it, since it's a new TODO.

Reply:
Not yet... I propose to leave it as a very low priority TODO in the code rather than opening an issue for it, because it is not a blocker for us.